spec/requests/api/v1/people_controller_index_spec.rb

```diff
@@ -32,5 +32,21 @@
         expect(People::Finder).to have_received(:new).with(police_national_computer: 'AB/1234567')
       end
     end
+
+    context 'when Nomis offender number no is provided' do
+      let(:nomis_offender_no) { 'G5033UT' }
+      let(:params) { { filter: { nomis_offender_no: 'G5033UT' } } }
+      let(:people_finder) { instance_double('People::Finder', call: Person.all) }
+
+      it 'requests data from NOMIS', with_json_schema: true do
+        allow(People::Finder).to receive(:new).and_return(people_finder)
+        allow(Moves::ImportPeople).to receive(:new).with([person_nomis_prison_number: nomis_offender_no])
```

Comment:
You're preparing the test here, you can extract this in a before block like in the other tests you're committing. The before filter and the context look like a very good match too. My 0.2.

Reply:
moved to before block